docs/using.rst

```diff
@@ -119,7 +119,7 @@ For more information about the layout
 of the ``/etc/letsencrypt`` directory, see :ref:`where-certs`.
 
 .. _Docker: https://docker.com
-.. _`install Docker`: https://docs.docker.com/userguide/
+.. _`install Docker`: https://docs.docker.com/engine/installation/
```

Comment:
That's what I did in bbc529b, but since the purpose of this link is to tell people how to install Docker, I added another commit linking to the installation section of their documentation. I don't believe they had a separate section like this when we first wrote these instructions.